### DISCUSSION TITLE
Request to Unlink University Email from Restricted GitHub Account

### DISCUSSION THREAD

**Discussion by Ahmed-Islam-AI** [community/community#136290] [GitHub Education]
### Hi everyone,

I am facing an issue with my university mail
I have attached my university mail with my previous GitHub account but that GitHub account is restricted by GitHub now when I attach the same mail with my current GitHub account it shows me that it is already in use!!!

how can I disconnect my mail with the previous account.....like is there any option in the mail so that I can disconnect it

and apply for github campus export and get github student resources

**Comment by pierrelouisguichard** [comment]
Hey,

The best approach is to reach out to GitHub Support directly. They should be able to help you disassociate the email from the restricted account so you can use it with your new account.

Best of luck!

**Reply by Ahmed-Islam-AI** [reply]
Thank you but i have already sent a mail but did not get a valid response from GitHub customer support team... they said will contact you so and still i am waiting.....today i have created another ticket let see what will happen this time.

**Comment by davevad93** [comment]
Hi @Ahmed-Islam-AI ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

However, if you have already applied for GitHub Education benefits with your old account, you'll not be able to request them again with your current account, because the benefits are linked to that email and aren't transferable.

**Reply by Ahmed-Islam-AI** [reply]
unfortunately, I applied for the github student pack and was selected for the GitHub pack on my previous account but at that time i got a suspending account email without doing any violence.....let see i have created another ticket...
